data/korath/korath missions.txt

```diff
@@ -257,3 +270,207 @@ mission "Small Scale Delivery"
+			`	"The most advanced group of surviving Korath lives in systems near yours, according to pirate data. It seems the Korath have been raiding ships in pirate and Syndicate systems for food, while the pirates have been pillaging Korath ships for tech. I'm not sure why the Korath have limited themselves to piracy. They certainly have the firepower to dominate a few dozen of your planets.`
+			`	"From our race's records, it is quite unlike the Korath to be so close to a more primitive race without making demands. Their resources may be committed to another war, or they may be building up supplies for a shock and awe style attack. It's anyone's guess at this point.`
+			`	"Even the Unfettered are concerned about this. A few dozen human planets under the dominion of the Korath could make them a force to be reckoned with again. They could even be a threat to the Hai."`
+			choice
+				`	"Sounds foreboding."`
+					goto foreboding
+				`	"When do I get my cut of the spoils?"`
+			`	"Ha! I'd think you were a True Hai if it weren't for the missing fur. Don't worry, Captain, I'll make sure the others give you a suitable payment."`
+			`	Batu sighs as Choot'k continues. "We've made an unpleasant discovery recently. The Korath have stealth drones hiding in two pirate systems: Alcyone and Almach. They've been using these to spy on nearby traffic, including us. It's also... eh, Batu?"`
+				goto next
+			label foreboding
+			`	"Indeed it does, especially with our recent discovery. The Korath have stealth drones hiding in two pirate systems: Alcyone and Almach. They've been using these to spy on nearby traffic, including us. It's also... eh, Batu?"`
+			label next
+			`	Batu nods and pulls up something on his datapad. He reads mechanically to you. "The drone fleet acts as a phase array transceiver and hyperwave sounder. A phase array uses signals from multiple sources (the drones) transmitting or receiving slightly out of phase, to emulate a much larger antenna. It's the same principle as phase array radars used for meteorology and phase array telescopes for astronomy. Each drone's communication system is not much stronger than a human hyperspace relay, but the fleet acts together as one powerful relay. We think it's able to relay signals as far as 120 light-years, without the need for starlanes. What's most concerning is that relaying is only a secondary functionality; they're mostly observation systems.`
+			`	He slides his finger across the pad and continues. "They were hard to detect, and have probably been hiding for years, perhaps decades. From debris analysis, we determined the drones are painted with a material that tricks human sensors, so seeing them is tough. Their transmissions are cleverly disguised as ambient noise, so you can't listen for them either. However, we've developed computer algorithms that'll let you find them. Essentially, your computer finds where they aren't, and works inward from there." He looks up from his datapad at last.`
```

Comment:
```suggestion
			`	He slides his finger across the pad and continues. "They were hard to detect, and have probably been hiding for years, perhaps decades. From debris analysis, we determined the drones are painted with a material that tricks human sensors, so seeing them is tough. Their transmissions are cleverly disguised as ambient noise, so you can't listen for them either. However, we've developed computer algorithms that'll let you find them. Essentially, your computer finds where it can be sure they aren't and works through a process of elimination from there." He looks up from his datapad at last.`
```
Something like this, making it more explicit could help with the explaining meaningfully as well, I think.

Reply:
I reworked those two sentences, and included a game developer inside joke. The algorithm uses a quad tree, which is a data structure for searching 2D space. You'd need an octree to search 3D space, so this is an inside joke about combat in Endless Sky being 2D.